sdk/python/kfp/test_utils.py

```diff
@@ -0,0 +1,58 @@
+from kfp.dsl import ContainerOp
+from utils import use_secret
+import unittest
+import inspect
+
+
+class TestAddSecrets(unittest.TestCase):
+
+    def test_use_default_use_secret(self):
+        op1 = ContainerOp(name='op1', image='image')
+        secret_name = "my-secret"
```

Comment:
nit: usage of quote should be consistent, i.e., always use `'` or `"` unless definitely needed